src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -83,6 +83,13 @@ public inherited sharing class RD2_OpportunityEvaluationService {
         } set;
     }
 
+    /**
+     * @description Indicates if Recurring Donation Settings close action on Open Opportunities 
+     * is overriden when Recurring Donation is Closed
+     */
+    public static Boolean isCloseActionOverride = false;
```

Comment:
What would you think of specifying what the override is for the var name? Maybe `isDoNotVoidOpenOpps` or `overrideVoidOfOpenOpps`?